lib/omnibus/packagers/bff.rb

```diff
@@ -229,6 +232,9 @@ def create_bff_file
       FileSyncer.glob(File.join(staging_dir, 'tmp/*.bff')).each do |bff|
         copy_file(bff, File.join(Config.package_dir, create_bff_file_name))
       end
+
+      # chown back to original user's uid/gid so cleanup works correctly
+      shellout!("sudo chown -Rh #{original_uid}:#{original_gid} #{staging_dir}")
```

Comment:
So I've been bothered by our use of `sudo` in this packager from the start. I wonder if we should re-examine our use of `mkinstallp` and just have the packager perform the steps as mentioned here:
http://compgroups.net/comp.unix.aix/why-does-mkinstallp-insist-on-being-run-as-root/1250985

If not I think we should ensure we always chown things back.